charts/metering-ansible-operator/upstream-values.yaml

```diff
@@ -49,6 +49,41 @@ olm:
     version: "4.2.0"
     minKubeVersion: "1.11.0"
 
+    description: |
+      Operator Metering is a chargeback and reporting tool to provide accountability for how resources are used across a cluster. Cluster admins can schedule reports based on historical usage data by Pod, Namespace, and Cluster wide. Operator Metering is part of the [Operator Framework](https://coreos.com/blog/introducing-operator-framework-metering).
+
+      Read the user guide for more details on [running and viewing your first report](https://github.com/operator-framework/operator-metering/blob/master/Documentation/using-metering.md).
+
+      ### Core capabilities
+
+      * **Chargeback/Showback** - Break down the reserved and utlized resources requested by applications.
+
+      * **Pod, Namespace & Cluster Reports** - Built in reports exist to break down CPU and RAM in any way you desire.
+
+      * **Scheduled Reports** - Schedule reports to run on a standard interval, eg. daily or monthly
+
+      * **Post-Processing** - Reports are generated in CSV format and stored in persistent storage for further post-processing. Use this to send reminder emails, integrate into your ERP system, or graph on a dashboard.
+
+      * **HTTP API** - Reports can be queried from an in-cluster HTTP API in addition to reading from persistent storage.
+
+      ### Before you start
+
+      Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 4 CPU cores. At least one Node should have 2GB of RAM and 2 CPU cores. Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+
+      ### Common Configurations
+
+      Metering works out of the box without any customizations or configuration. Read the [full documentation](https://github.com/operator-framework/operator-metering/blob/master/Documentation/metering-config.md) for more details.
```

Comment:
Do you want to still link to GH here? If not, I'd link to About Metering:
https://docs.openshift.com/container-platform/4.2/metering/metering-about-metering.html

Reply:
nvm, I see this is the upstream values.